lab-ron/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This looks good, please remove the "scripts" > "test" property since you don't have any mocha tests, alternatively specify a command which says you don't have any tests.  Ex: 'echo "No tests to perform."'